Tasks/AzureResourceGroupDeploymentV2/models/DeployAzureRG.ts

```diff
@@ -104,6 +105,7 @@ export class AzureRGTaskParameters {
             var endpointTelemetry = '{"endpointId":"' + connectedService + '"}';
             console.log("##vso[telemetry.publish area=TaskEndpointId;feature=AzureResourceGroupDeployment]" + endpointTelemetry);
             this.subscriptionId = tl.getEndpointDataParameter(connectedService, "SubscriptionId", true);
+            this.endpointPortalUrl = tl.getEndpointDataParameter(connectedService, "armManagementPortalUrl", true);
```

Comment:
Will this value be available on endpoints which were declared way before management group came into the picture? You've marked this value as required if this value isn't there it would break at this point.

Reply:
Endpoint Portal URL is not related to Management Groups. It's related to the environment.